docs/datasets/index.rst

```diff
@@ -52,10 +52,83 @@ Note that in Gammapy, 2D image analyses are done with 3D cubes with a single
 energy bin, e.g. for modeling and fitting,
 see the `2D map analysis tutorial <./tutorials/image_analysis.html>`__.
 
+To analyse multiple runs, you can either stack the datasets together, or perform
+a joint fit across multiple datasets.
 
-Getting Started
-===============
+Stacking Multiple Datasets
+==========================
 
+Stacking datasets implies that the counts, background and reduced IRFs from all the
+runs are binned together to get one final dataset for which a likelihood is
+computed during the fit.
+
+Counts, background and exposure (lying outside the masked regions are simply summed),
```

Comment:
I think closing parentheses are not set correctly here. In general my proposal would be to only use terms like "region defined by the `mask_safe`" or "data range defined by the `mask_safe`" and not mix it with inside / outside exclusion regions (see later in the table as well), because typically exclusion regions are handled independently from the safe data range.